biz.aQute.repository/test/aQute/bnd/repository/p2/provider/P2RepositoryTest.java

```diff
@@ -42,19 +44,47 @@ public void testSimple() throws Exception {
 		}
 	}
 
-	// @Test does not work for some reason
+	@Test
 	public void testXtext() throws Exception {
+
 		try (P2Repository p2r = new P2Repository()) {
 			Workspace w = Workspace.createStandaloneWorkspace(new Processor(), tmp.toURI());
+			w.addBasicPlugin(new ProgressPlugin() {
```

Comment:
Is this plugin all just for debugging since it does not test anything. If it is just for test debugging, we should probably not leave it in.